File: benchmarkDS/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Lse5.cpp

#include <iostream>
#include <string>
#include <cstdlib>

std::string getElementAtIndex(int index) {
    std::string elements[] = {"apple", "banana", "cherry", "date", "elderberry"};
    int size = 5;
    
    if (index >= 0 && index < size) {
        return elements[index];
    } else {
        return "Index out of bounds";
    }
}

int main(int argc, char* argv[]) {
    // Test cases
    std::cout << "Test case 1: " << getElementAtIndex(0) << std::endl;
    std::cout << "Test case 2: " << getElementAtIndex(2) << std::endl;
    std::cout << "Test case 3: " << getElementAtIndex(4) << std::endl;
    std::cout << "Test case 4: " << getElementAtIndex(-1) << std::endl;
    std::cout << "Test case 5: " << getElementAtIndex(10) << std::endl;
    
    // Command line argument handling
    if (argc > 1) {
        int index = std::atoi(argv[1]);
        std::cout << "Element at index " << index << ": " << getElementAtIndex(index) << std::endl;
    }
    
    return 0;
}

File: benchmarkDS/ZeroShot/CLAUDE_claude-sonnet-4-5-20250929/Lse5.java

public class Lse5 {
    public static String getElementAtIndex(int index) {
        String[] elements = {"apple", "banana", "cherry", "date", "elderberry"};
        
        if (index < 0 || index >= elements.length) {
            return "Error: Index out of bounds";
        }
        
        return elements[index];
    }
    
    public static void main(String[] args) {
        // Test cases
        System.out.println("Test Case 1: " + getElementAtIndex(0));
        System.out.println("Test Case 2: " + getElementAtIndex(2));
        System.out.println("Test Case 3: " + getElementAtIndex(4));
        System.out.println("Test Case 4: " + getElementAtIndex(-1));
        System.out.println("Test Case 5: " + getElementAtIndex(10));
        
        // Command line argument processing
        if (args.length > 0) {
            try {
                int index = Integer.parseInt(args[0]);
                System.out.println("\\nCommand line result: " + getElementAtIndex(index));
            } catch (NumberFormatException e) {
                System.out.println("Error: Invalid number format");
            }
        }
    }
}
